logos/README.md

```
@@ -63,16 +64,22 @@ To deploy Logos locally or on a server:
-   
-   ```
-   docker compose -f ./logos/docker-compose.yaml build
-   ```
-   
-   and afterwards
-   
+   Logos provides two docker-compose configurations:
+
+   | File | Purpose |
+   |------|---------|
+   | `docker-compose.yaml` | **Production** - pulls pre-built images from GitHub Container Registry |
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe does not align with header for style "aligned"

(MD060, table-column-style)